documentation/projects/proposals/nuxt_3_migration/20230608-implementation_plan_nuxt_3_migration.md

```diff
@@ -0,0 +1,277 @@
+| `"@nuxtjs/composition-api/module"` | Delete - natively supported by Nuxt 3                                                                |                                                                                                                                                          |
+| `"portal-vue/nuxt"`                | Delete - natively supported by Vue 3 / Nuxt 3                                                        |                                                                                                                                                          |
+| `"cookie-universal-nuxt"`          | Delete - natively supported by Nuxt 3, [useCookie](https://nuxt.com/docs/api/composables/use-cookie) |                                                                                                                                                          |
+| `"@nuxtjs/svg-sprite"`             | Update to current that supports Nuxt 3                                                               |                                                                                                                                                          |
+| `"@nuxtjs/eslint-module"`          | Update to current that supports Nuxt 3                                                               | https://github.com/nuxt-modules/eslint                                                                                                                   |
+| `"@pinia/nuxt"`                    | Update to current version that supports Nuxt 3                                                       |                                                                                                                                                          |
+| `"@nuxtjs/proxy"`                  | Replace with Nitro routeRules or use `proxyRequest`                                                  | https://stackoverflow.com/questions/76120894/how-do-i-set-proxy-with-baseurl-with-nuxt3-and-nitro-config, https://github.com/nuxt/nuxt/discussions/15907 |
+| `"@nuxtjs/redirect-module"`        | Define Nitro routeRules in `nuxt.config`                                                             | https://nitro.unjs.io/config/#routerules                                                                                                                 |
+| `"@nuxtjs/sentry"`                 | No official support                                                                                  | Workaround in https://github.com/nuxt-community/sentry-module/issues/530                                                                                 |
+| `"vue-plausible"`                  | Update to current version that supports Nuxt 3 or https://github.com/nuxt-modules/plausible          | Note: enabling outbound tracking breaks links with target blank in `vue-plausible`                                                                       |
```

Comment:
Would each of these that need workarounds be separate issues? I've only looked at the proxy one but it looks certainly complex enough to be its own issue. It also looks like we could implement the workaround for it in Nuxt 2, FWIW, perhaps in conjunction with #2392 which would complicate the current configuration anyway.

Reply:
Yes, separating the workarounds into separate issues is a good idea. Adding modules can also be worked on in parallel.